addons/payment_stripe/controllers/main.py

```diff
@@ -6,7 +6,6 @@
 import logging
 import pprint
 from datetime import datetime
-
```

Comment:
The blank line should stay, as `werkzeug` is a third-party library unlike the other imports.